jedeschule/utils/wfs_basic_parsers.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+from scrapy.http import Response
+
+
+def parse_geojson_features(response: Response):
```

Comment:
I think what you could do instead of (only?) exposing the function here is to write a new spider (kind of like we do for `school_spider.py` but with an actual implementation of the `parse` method) and then make the actual spiders inherit from that (so that it would e.g. read `class BerlinSpider(SchoolSpider, GeoJsonSpider):`). I feel like this might be more common in Scrapy but it also works just as fine your way. Just curious about your thoughts.